### DISCUSSION TITLE
string_l contains string_r comparisons

### DISCUSSION THREAD

**Discussion by mickey-liu** [moj-analytical-services/splink#1360] [Q&A]
Hi 

I was wondering if there are out of the box comparisons I can use to check whether a string exists in another string for Spark linking.

i.e.
```
"comparisons": [
    cl.exact_match("name"),
    cl.string_contains(col_string_left, col_string_right),
    ...
]
```
Thanks

**Comment by RobinL** [comment]
Hiya.  There's no pre-configured comparison for this, but you can define it yourself using option 4 [here](https://moj-analytical-services.github.io/splink/topic_guides/customising_comparisons.html#method-4-providing-the-spec-as-a-dictionary)
